samples/snapshot/pom.xml

```diff
@@ -48,11 +48,13 @@
       <version>0.1.7-SNAPSHOT</version>
     </dependency>
     <!-- {x-version-update-end} -->
+    <!-- {x-version-update-start:google-cloud-pubsub:released} -->
```

Comment:
This refers to cloud pubsub not pubsublite. Is this intentional?

Reply:
This tag works by picking up version numbers from `versions.txt`. 
I'm guessing this is not needed because renovate bot is supposed to update deps in POMs. I may need to close this PR, but I will wait for an answer from Jeff.